apps/protocol-frontend/src/components/ContributionsTable.tsx

```diff
@@ -1,25 +1,26 @@
-import { useState, useEffect, useMemo } from 'react';
+import { useEffect, useMemo, useState } from 'react';
 import {
-  Table,
+  Box,
+  chakra,
   HStack,
   IconButton,
+  Stack,
+  Table,
```

Comment:
Looks like <Tooltip/> needs to be added again. I think it was in before but not sure how it was removed -- don't think it was in these changes though but we can import it from chakra